app/services/update_user_password.rb

```diff
@@ -0,0 +1,47 @@
+class UpdateUserPassword
+  delegate :recovery_code, to: :encryptor
+
+  def initialize(user, user_session, password)
+    @user = user
+    @user_session = user_session
+    @password = password
+  end
+
+  def call
+    form = UpdateUserPasswordForm.new(user)
+    form.submit(password)
+    success = form.valid?
+    process_valid_submission if success
+    FormResponse.new(success: success, errors: form.errors.messages)
```

Comment:
I don't follow. This is what we've been doing with all classes that controllers interact it. How would you suggest we return the FormResponse?